_clanky/výběr-katalogizačního-systému-pro-město-Brno.md

```diff
@@ -1,178 +1,83 @@
-
-Další nezanedbatelnou výhodou je, že HUB umožňuje postavit si nad katalogem pokročilou webovou nadstavbu bez potřeby jakýchkoliv IT zručností. Web je tvořen pomocí série nástrojů založených na principu WYSIWYG (what you see is what you get), což umožňuje nejen jeho rychlé a jednoduché sestavení, ale i správu.  
-
-
-Obr. č.5: Editační prostředí pro tvorbu webové nadstavby.
-
-
-Tak jako každé jiné řešení ani ArcGIS HUB není dokonalý a má své nevýhody - problémy s dvojjazyčností, menší možnosti co se týče neprostorových formátů dat nebo problémy s nekompatibilní verzí standardu DCAT-AP stejně jako u CKANu. ArcGIS HUB také není vhodným řešením pro města, které nemají podstatnou část svých dat ve službách nebo geodatabázích postavených nad architekturou od ESRI.
-
-Pro Brno bylo však toto řešení ideální, jelikož funkcionality a pokročilé technologické řešení popsané výše, umožňují poskytovat veřejnosti kvalitní, přesné a aktuální otevřená data bez nutnosti vynaložení velkých finančních prostředků. Kromě Brna, jako jediného většího města v ČR, využívá v současnosti HUB pro publikaci open dat ještě Agentura ochrany přírody a krajiny. V zahraničí je užívání HUBu mnohem rozšířenější - využívají jej například města jako Sydney, Washington D.C. nebo Pařížská metropolitní oblast.
-
-
-
-
-
-
-
-Obr. č.1: Custom-built portál otevřených dat města Ostrava.
-
-
-
+[PRHCKN]: http://https://opendata.praha.eu/ "CKAN"
+[GOLEMIO]: https://golemio.cz/ "Golemio"
+[GEOPORTAL]: https://https://www.geoportalpraha.cz/cs/data/otevrena-data/seznam "Geoportál"
+[OSTRAVU]: https://opendata.ostrava.cz/ "Ostravu"
+[PLZEN1]: https://opendata.plzen.eu/ "Plzeň"
+[CKAN]: https://ckan.org/ "CKAN"
+[GLASGOW]: http://data.glasgow.gov.uk/dataset?q= "Glasgow"
+[KANADSKA]: https://open.canada.ca/en/open-data "kanadská vláda"
+[PLZEN2]: https://opendata.plzen.eu/ "Plzeň"
+[OLOMOUC]: https://data.olomouc.eu/ "Olomouc"
+[HRADEC]: http://opendata.mmhk.cz/ "Hradec Králové"
+[NKOD]: https://data.gov.cz/datov%C3%A9-sady?dotaz= "Národní katalog otevřených dat"
+[OFN1]: https://ofn.gov.cz/rozhran%C3%AD-katalog%C5%AF-otev%C5%99en%C3%BDch-dat/2019-04-04/ "Otevřenou formální normu"
+[ENTRYSCAPE]: https://entryscape.com/en/ "Entryscape"
+[DATAPRESS]: (https://datapress.com/ "DataPress"
+[DATAWORLD]: https://data.world/ "Data.World"
+[ARCGISHUB]: https://www.esri.com/en-us/arcgis/products/arcgis-hub/overview "ArcGIS HUB"
+[DATABRNO]: (http://data.brno.cz/ "data.brno.cz"
+[WAZE]: https://data.brno.cz/search?collection=Dataset&q=waze "data například z aplikace Waze"
+[OFN2]: https://ofn.gov.cz/rozhran%C3%AD-katalog%C5%AF-otev%C5%99en%C3%BDch-dat/2019-04-04/ "OFN rozhraní katalogů otevřených dat"
```

Comment:
@spalrobert  Tadyta URL chceme neencodovaná

Reply:
Ahoj je to tak ako si myslím že by tam nemali byť url ktoré majú namiesto skutočných znakov náhradu ako napr. % ako je tu napríklad tu: https://ofn.gov.cz/rozhran%C3%AD-katalog%C5%AF-otev%C5%99en%C3%BDch-dat/2019-04-04/

V majle mi prišlo cca 15 url že mám opraviť a viaceré ale nevidím ako by sa mali zmeniť ako napr. https://opendata.plzen.eu/ a pritom https://opendata.ostrava.cz/ nie je medzi linkami treba zmeniť.

Kontroloval som aj iné články ako tam majú linky a neviem nájsť rozdiel.

Jediný je s týmy %. Mohol by si mi to trocha objasniť čo myslíš? Díky!

Reply:
Ano, URL neencodovat pomocí %.

Nevím o kterém mailu teď mluvíš.

Reply:
Encodovanie som opravil a v majle mi prišiel komentár že kodovanie je zle ku asi 15 urlkam tak ma to zmiatlo trocha pretože tam boli aj url ktoré su ok. Asi to posiela nejak v batchit podľa commitov alebo čo.